docker-for-mac/install.md

```diff
@@ -1,6 +1,6 @@
 ---
 description: How to install Docker Desktop on Mac
-keywords: mac, beta, alpha, install, download
+keywords: mac, install, download, run, docker, local, machine
```

Comment:
do we have a different page that will win download instead?

Reply:
Do you mean a different page where we provide Desktop downloads?

Reply:
As discussed other pages that are better suited have the install tag